image-inspector.go

```diff
@@ -58,6 +62,46 @@ type ImageInspectorOptions struct {
 	// PasswordFile is the location of the file containing the password for authentication to the
 	// docker registry.
 	PasswordFile string
+	// Oscap controls whether or not a OpenSCAP is excuted on the served image.
+	Oscap bool
+}
+
+type OpenSCAPStatus string
+
+const (
+	StatusNotRequested OpenSCAPStatus = "not-requested"
+	StatusSuccess      OpenSCAPStatus = "Success"
+	StatusError        OpenSCAPStatus = "Error"
```

Comment:
@moolitayer I jumped to fix this too fast but now that I look at my origin [1] and I see that in kubernetes they are using CamelCase for these constants so I am reverting back and will just change ```"not-requested"``` to ```"NotRequested"```